### PR TITLE
Add more test cases for policy environment 2023

### LINKED ISSUE #447: Add more test cases for policy environment 2023
### Current and desired situation

#440, #433, #442 implemented several updates for 2023. They include only insufficient test cases, as online calculators are typically not yet updated and some reforms like Bürgergeld law is not even passed yet. 

### Proposed implementation

Get back to the implemented changes in spring/summer 2023 and:

- Check whether there are any changes (e.g. with respect to Bürgergeld)
- Update test cases in the PR's above if necessary.
- Add test cases for all parameters that have been updated in the parameters above.

### PR DESCRIPTION
**Pull request by LauraGergeleit**
### What problem do you want to solve?

Closes #447. Double-checked, updated and added testcases for implemented updates for 2023 (#440, #442), now that online calculators are mostly updated.

### CODE REVIEW

File: src/_gettsim_tests/test_data/kindergeld.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-hh_id,tu_id,p_id,alter,arbeitsstunden_w,kind,in_ausbildung,bruttolohn_m,_zu_verst_eink_ohne_kinderfreib_tu,jahr,kindergeld_m_tu,kinderbonus_m_tu,kindergeld_m_hh,kinderbonus_m_hh,kinderbonus_m_tu,note
+hh_id,tu_id,p_id,alter,arbeitsstunden_w,kind,in_ausbildung,bruttolohn_m,_zu_verst_eink_ohne_kinderfreib_tu,jahr,kindergeld_m_tu,kinderbonus_m_tu,kindergeld_m_hh,kinderbonus_m_hh,kinderbonus_m_tu_1,note
```
Comment (ChristianZimpelmann) [inline src/_gettsim_tests/test_data/kindergeld.csv:1]:
```suggestion
hh_id,tu_id,p_id,alter,arbeitsstunden_w,kind,in_ausbildung,bruttolohn_m,_zu_verst_eink_ohne_kinderfreib_tu,jahr,kindergeld_m_tu,kinderbonus_m_tu,kindergeld_m_hh,kinderbonus_m_hh,kinderbonus_m_tu,note
```

File: src/_gettsim_tests/test_data/soli_st.csv [hunk 2]
```diff
@@ -1,7 +1,8 @@
-hh_id,tu_id,p_id,kind,abgelt_st_tu,eink_st_mit_kinderfreib_tu,,jahr,,soli_st_tu
+hh_id,tu_id,p_id,kind,abgelt_st_tu,eink_st_mit_kinderfreib_tu,Column1,jahr,_1,soli_st_tu
```
Comment (ChristianZimpelmann) [inline src/_gettsim_tests/test_data/soli_st.csv:1]: What is this change about? Probably better to delete the empty columns instead.

**Review by ChristianZimpelmann — APPROVED** [review]
Great work, thanks!

Just two small requests.

Additionally, can you put together the test files which haven't been updated to 2023, yet?

**Comment by LauraGergeleit** [comment]
> Additionally, can you put together the test files which haven't been updated to 2023, yet?

The following test files do not include test cases in 2023:
arbeitsl_geld
benefit_checks
elterngeld
favorability_check
full_taxes_and_transfers
grundrente
grunds_im_alter
renten_alter
renten_anspr
sozialv_beitr
unterhaltsvors
zu_verst_eink

File: src/_gettsim_tests/test_data/kindergeld.csv [hunk 3]
```diff
@@ -1,4 +1,4 @@
-hh_id,tu_id,p_id,alter,arbeitsstunden_w,kind,in_ausbildung,bruttolohn_m,_zu_verst_eink_ohne_kinderfreib_tu,jahr,kindergeld_m_tu,kinderbonus_m_tu,kindergeld_m_hh,kinderbonus_m_hh,kinderbonus_m_tu,note
+hh_id,tu_id,p_id,alter,arbeitsstunden_w,kind,in_ausbildung,bruttolohn_m,_zu_verst_eink_ohne_kinderfreib_tu,jahr,kindergeld_m_tu,kinderbonus_m_tu,kindergeld_m_hh,kinderbonus_m_hh,kinderbonus_m_tu2,note
```
Comment (hmgaudecker) [inline src/_gettsim_tests/test_data/kindergeld.csv:1]:
```suggestion
hh_id,tu_id,p_id,alter,arbeitsstunden_w,kind,in_ausbildung,bruttolohn_m,_zu_verst_eink_ohne_kinderfreib_tu,jahr,kindergeld_m_tu,kinderbonus_m_tu,kindergeld_m_hh,kinderbonus_m_hh,kinderbonus_m_tu,note
```

?

Or was that on purpose?

**Review by hmgaudecker — COMMENTED** [review]
Excellent, thank you! 

Just one minor thing left, not sure whether a typo or on purpose.

File: src/_gettsim_tests/test_data/kindergeld.csv (hunk 3 above)
Comment (LauraGergeleit) [reply src/_gettsim_tests/test_data/kindergeld.csv:1]: No, that was not on purpose, that must have been changed automatically, why I don't know.. But I will correct it!

Comment (hmgaudecker) [reply src/_gettsim_tests/test_data/kindergeld.csv]: Excellent, thank you!

**Pull request merged**